src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Properties/Resources.resx

```diff
@@ -433,6 +433,9 @@
     <value>Cellular and SIM</value>
     <comment>Area NetworkAndInternet</comment>
   </data>
+  <data name="ChangePrograms" xml:space="preserve">
+    <value>Change programs</value>
```

Comment:
nit: this and "Repair programs" might be overkill. but, more the better I guess..

Reply:
This are only variant to search for. They are only shown in tool tip.